lib/datasource/artifactory/index.spec.ts

```diff
@@ -0,0 +1,154 @@
+import { getPkgReleases } from '..';
+import * as httpMock from '../../../test/http-mock';
+import { loadFixture } from '../../../test/util';
+import { EXTERNAL_HOST_ERROR } from '../../constants/error-messages';
+import { logger } from '../../logger';
+import { joinUrlParts } from '../../util/url';
+import { ArtifactoryDatasource } from '.';
+
+const datasource = ArtifactoryDatasource.id;
+
+const testRegistryUrl = 'https://jfrog.company.com/artifactory';
+const testLookupName = 'project';
+const testConfig = {
+  registryUrls: [testRegistryUrl],
+  depName: testLookupName,
+};
+
+function getPath(folder: string): string {
+  return '/' + folder;
+}
+
+describe('datasource/artifactory/index', () => {
+  beforeEach(() => {
+    jest.resetAllMocks();
+  });
+
+  describe('getReleases', () => {
+    it('parses real data (folders): with slash at the end', async () => {
+      httpMock
+        .scope(testRegistryUrl)
+        .get(getPath(testLookupName))
+        .reply(200, loadFixture('releases-as-folders.html'));
```

Comment:
Can you use module const for fixtures, so they don't need to be reloaded multiple times